packages/manager/modules/navbar/src/notifications-menu/service.js

```diff
@@ -12,13 +14,15 @@ export default class Notifications {
     atInternet,
     OvhApiNotificationAapi,
     ovhManagerNavbarMenuHeaderBuilder,
+    TranslateService,
   ) {
     this.$interval = $interval;
     this.$q = $q;
     this.$translate = $translate;
     this.atInternet = atInternet;
     this.OvhApiNotificationAapi = OvhApiNotificationAapi;
     this.NavbarBuilder = ovhManagerNavbarMenuHeaderBuilder;
+    moment.locale(TranslateService.getUserLocale());
```

Comment:
Be careful, it will change `moment` locale for global.

It's done in each applications (examples : [Web](https://github.com/ovh/manager/blob/master/packages/manager/apps/web/client/app/app.js#L491), [Public cloud](https://github.com/ovh/manager/blob/master/packages/manager/apps/public-cloud/src/index.js#L79))

I've added it in hub in another PR : https://github.com/ovh/manager/commit/9a187809327f9ca1cba470f721c5b10db5159c7b#diff-c384e559dbea2c7fddd21b70ef116886R46